src/Avalonia.Controls/Converters/PlatformKeyGestureConverter.cs

```diff
@@ -148,6 +148,8 @@ private static string ToString(Key key)
                 Key.D9 => "9",
                 Key.Decimal => ".",
                 Key.Divide => "/",
+                Key.Down => "Down Arrow",
```

Comment:
Why are we using text here instead of symbols?

Reply:
Because as far as we could tell, on Windows at last, menus use e.g. `Ctrl+Up Arrow` as the key gesture text.